src/App.jsx

```diff
@@ -1,8 +1,8 @@
 import { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Route, Routes, useLocation } from 'react-router-dom';
-import Navigation from './components/navigation';
 import ProtectedRoute from './components/ProtectedRoute';
+import Navigation from './components/navigation';
```

Comment:
Would it make sense not to change this order? I ran the import organizer from VSCode and it reverted this change.